internal/template-validator/webhooks/utils.go

```diff
@@ -0,0 +1,98 @@
+package validating
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	_ "github.com/go-kit/kit/log"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubevirt "kubevirt.io/client-go/api/v1"
```

Comment:
I have noticed that these imports are different from the ones in `kubevirt-template-validator` repository. They are ok, but did you change anything else from the original code?

Reply:
For this specific file, I included the changes from https://github.com/kubevirt/kubevirt-template-validator/pull/60
But not anything in other files